example/huge-tasks/dataset.py

```diff
@@ -0,0 +1,25 @@
+from starwhale import dataset
+from starwhale.utils.debug import console
+
+try:
+    from .utils import random_text
+except ImportError:
+    from utils import random_text
+
+
+def build():
+    with dataset("huge-tasks-random-text-1m") as ds:
+        console.print(f"start to build dataset: {ds.uri.name}")
+        for i in range(0, 1000 * 1000 - 1):
```

Comment:
The rows number of range(0, 999999) is 999999, remove the '-1'?

Reply:
Only wants 999999 rows.

range(0, 100) --> 100 rows
range(0, 100-1) --> 99 rows